spec/e2e/accessibility_spec.js

```diff
@@ -29,6 +49,22 @@ describe('accessibility', () => {
         .exclude('.footer-nav') // See: LG-4038 (TODO: Remove with implementation of LG-4038)
         .analyze();
       expect(results).toHaveNoViolations();
+
+      /** @type {SimplifiedLink[]} */
+      let links = await page.evaluate(() =>
+        Array.prototype.map.call(document.querySelectorAll('a'), a => {
+          // There's a weird serialization boundary here so we encode the links as a structure
+          return {
+            innerText: a.innerText.trim(),
+            href: a.href,
+            className: a.className,
+            target: a.target,
+          };
+        })
+      );
+      Array.from(links).forEach(a => {
+        expect(a).toNotHaveTargetBlank();
```

Comment:
@aduth would love some opinions on how to organize these additional assertions

- Should I move the whole "page.evaluate" bit inside a helper function?
- Should I move these into a helper file and import them?

Reply:
I ended up moving stuff into a separate file in 822861d423e37f827e2b1d74a24a228792138e12